### PR TITLE
Feature:switch component

### PR DESCRIPTION
**Pull request by tienviet99**

### CODE REVIEW

File: src/components/switch/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styles from "./switch.module.css";
+
+interface SwitchProps {
+  onClickSwitch: (status: boolean) => void;
+}
+
+export default function Switch({ onClickSwitch }: SwitchProps) {
+  const hanldeChangeSwitch = (e: any) => {
```
Comment (locbui95) [inline src/components/switch/index.tsx:8]: do not use any

File: src/components/switch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    onClickSwitch(e.target.checked);
+  };
+  return (
```
Comment (locbui95) [inline src/components/switch/index.tsx:11]: add space

File: src/components/switch/index.tsx (hunk 1 above)
Comment (locbui95) [inline src/components/switch/index.tsx:8]: wrong naming convention for handle event

Comment (tienviet99) [reply src/components/switch/index.tsx]: Done

File: src/components/switch/index.tsx (hunk 2 above)
Comment (tienviet99) [reply src/components/switch/index.tsx:13]: Done

File: src/components/switch/index.tsx (hunk 1 above)
Comment (tienviet99) [reply src/components/switch/index.tsx]: Done

**Review by locbui95 — APPROVED** [review]

**Pull request merged**